examples/getting-started-movielens/04-Triton-Inference-with-HugeCTR.ipynb

```diff
@@ -55,29 +55,42 @@
    "id": "71304a10",
```

Comment:
<p>you can say since we add <code>--model-control-mode=explicit</code> flag, the model wont be loaded at this step, we will load the model below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>done</p><p> </p>